src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -719,6 +719,7 @@ File 1
 True
 True
 True
+True
```

Comment:
The additional "True" appears because the input contains a property initializer "= 25" that was previously not instrumented.